client/notices/docs/example.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import ButtonGroup from 'components/button-group';
+import i18n from 'lib/mixins/i18n';
+import notices from 'notices';
+
+const NoticesGlobal = () => {
+	function callSuccessNotice() {
+		notices.success( i18n.translate( 'This is a global success notice' ) );
+	}
+
+	function callErrorNotice() {
+		notices.error( i18n.translate( 'This is a global error notice' ) );
+	}
+
+	function callInfoNotice() {
+		notices.info( i18n.translate( 'This is a global info notice' ) );
+	}
+
+	function callWarningNotice() {
+		notices.warning( i18n.translate( 'This is a global warning notice' ) );
+	}
+
+	return (
+		<div>
+			<h2>{ i18n.translate( 'Notices - Global' ) }</h2>
```

Comment:
Do we want to translate these strings? We don't translate [`Card` example](https://github.com/Automattic/wp-calypso/blob/master/client/components/card/docs/example.jsx), or any of our README files, and devdocs isn't enabled in production.